cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -148,6 +148,12 @@ func runApply(flags *applyFlags, userVersion string) error {
 		return errors.Wrap(err, "[upgrade/version] FATAL")
 	}
 
+	// block if the local etcd manifest is listening on local host only and the user explicitly opted out from etcd upgrade.
+	// this is necessary because we want all the user to move to the new etcd manifest with v1.14.
+	if cfg.Etcd.External == nil && etcdutil.IsEtcdLinsteningOnLocalHostOnly() && !flags.etcdUpgrade {
+		return errors.Errorf("kubeadm detected local etcd is still listening on local host only. Please upgrade etcd in order to avoid problem with new releases of kubeadm")
```

Comment:
Basically what @neolit123 already proposed, but with a slightly different message:
1. s/errors.Errorf/errors.New
2. `kubeadm detected that the local etcd member is still listening only on localhost. Please upgrade etcd to avoid problems with new releases of kubeadm`